src/index.ts

```diff
@@ -18,11 +42,14 @@ enum ColorType {
 }
 
 const rgbCallExpRegex =
-  /rgb\(\s*(\d{1,3}%?)\s*,\s*(\d{1,3}%?)\s*,\s*(\d{1,3}%?)\s*(,\s*0?\.\d+)?\)/;
+  /rgb\(\s*(\d{1,3}%?)\s*,?\s*(\d{1,3}%?)\s*,?\s*(\d{1,3}%?)\s*(,\s*0?\.\d+)?\)/;
```

Comment:
`rgb(255 255 255)` is valid

Reply:
Maybe we should do `(?:\s*,\s*|\s+)` instead, but no big deal